FEM/burgers.cpp

```diff
@@ -0,0 +1,134 @@
+#include "burgers.h"
+#include "PhysicalConstant.h"
+namespace Burgers
+{
+SolidBurgers::SolidBurgers(const Math_Group::Matrix* data)
```

Comment:
not important: you can pass a reference of `data` instead of a pointer. then you can simply write `data(0)`.